app/helpers/alchemy/pages_helper.rb

```diff
@@ -90,7 +90,7 @@ def render_menu(name, options = {})
         node_partial_name: "#{root_node.view_folder_name}/node"
       }.merge(options)
 
-      render(root_node, menu: root_node, node: root_node, options: options)
+      render(root_node.to_partial_path, menu: root_node, node: root_node, options: options)
```

Comment:
Interesting 🤔 I always thought this is the method Rails is calling internally anyways?

Reply:
Dito